src/theme/ods-chart-theme.ts

```diff
@@ -553,16 +577,12 @@ export class ODSChartsTheme {
    *
    * The generated tooltips or popover will implement the Orange Design System.
    *
-   * externalizePopover() needs:
-   * - popoverConfig: the configuration of the externalizePopover feature {@link ODSChartsPopoverConfig}
-   * - popoverDefinition: a renderer {@link ODSChartsPopoverDefinition} of the popover/tooltip
-   *
-   *   {@link ODSChartsPopoverManagers} gives preconfigured renderer
-   *
-   *   default value is  {@link ODSChartsPopoverManagers.NONE}: uses default Apache ECharts template to externalize tooltip/popover HTML element, implementing Orange Design system
-   *
-   * optionally you can use this call to set dataOptions
-   * @returns returns back the theme manager object
+   * @param popoverConfig the configuration of the externalizePopover feature {@link ODSChartsPopoverConfig}
+   * @param popoverDefinition renderer {@link ODSChartsPopoverDefinition} of the popover/tooltip. **{@link ODSChartsPopoverManagers}** is used to
+   * specify preconfigured renderer for Apache ECharts, Boosted 5 or Boosted 4 renderer.
```

Comment:
Add a warning that using Boosted 4 or Boosted 5 rendering requires dependency on the boosted library and access to the boosted global variable.